ui/config/targets.js

```diff
@@ -2,7 +2,7 @@
 
 const browsers = ['last 1 Chrome versions', 'last 1 Firefox versions', 'last 1 Safari versions'];
 
-const isCI = !!process.env.CI;
+const isCI = Boolean(process.env.CI);
 const isProduction = process.env.EMBER_ENV === 'production';
 
 if (isCI || isProduction) {
```

Comment:
sidenote: you could probably reduce your UI payload size by removing this (assuming you don't support IE11).

Reply:
Thanks! That's very good feedback! I'm not even sure what the Waypoint browser support policy is, but it should be fine to drop. I'll ask first and change it in another PR.